typescript/tsconfig.web.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "ES2015"
```

Comment:
We could target ESNext here, as we don't build with TypeScript. What do you think?

Reply:
Definitely! I didn't really know what do for this one so looking for more feedback for sure